sdk/eventgrid/Azure.Messaging.EventGrid.SystemEvents/tsp-location.yaml

```diff
@@ -1,3 +1,3 @@
 repo: Azure/azure-rest-api-specs
 directory: "specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents"
-commit: 90041e31d04e686567849f27d8641a5d61110508
+commit: b64ffae1370a03507b67e27502e7d108e81d0555
```

Comment:
Need to update to commit on main once https://github.com/Azure/azure-rest-api-specs/pull/35053 merges